sdk/test-utils/test-utils/src/chai.ts

```diff
@@ -0,0 +1,8 @@
+import chai from "chai";
+import { chaiAzure } from "./chaiAzure";
+import chaiAsPromised from "chai-as-promised";
```

Comment:
I'll look at what other plugins we use often and migrate them over. I also plan to add `assert.throwsAbortError` from keyvault...